docs/sdk/serverless-signing/combining-decryption-shares.md

```diff
@@ -4,99 +4,75 @@ import FeedbackComponent from "@site/src/pages/feedback.md";
-        value: '0',
-      },
-    },
-  ];
-  const message = 'Hello world';
-  const client = new LitNodeClient({
-    litNetwork: "datil-dev"
-  });
-  await client.connect();
-  const { ciphertext, dataToEncryptHash } = await LitJsSdk.encryptString(
-    {
-      accessControlConditions,
-      sessionSigs: {}, // your session
-      chain,
-      dataToEncrypt: message,
-    },
-    client
-  );
-
-  console.log("cipher text:", ciphertext, "hash:", dataToEncryptHash);
-```
-Let's break this down. The first step was creating your Access Control Condition (ACC), which is used to specify who or under what conditions your data should be able to be decrypted.
-
-The second step was actually encrypting the static content (string, file, zip, etc...) using the `encryptString` function. This returns a `ciphertext` and `dataToEncryptHash`. The `ciphertext`, `dataToEncryptHash`, chain data, and any other metadata (such as your `accessControlConditions`) should be stored on your storage provider of choice. A solid choice is IPFS. 
-
-## Using IPFS CID as an Access Control Parameter
-For this example, you can set your Access Control parameter as `currentActionIpfsId` which can be accomplished using the snippet below. This will mean that only a specific Lit Action (based on the IPFS CID where it has been deployed) will be able to decrypt your data. No other party will ever have access. This is useful for situations where you want to restrict access to sensitive information, like an API key, so that it can only be decrypted by a specific Lit Action.
-
-```js
-{
-  contractAddress: '',
-  standardContractType: '',
-  chain: 'ethereum',
-  method: '',
-  parameters: [':currentActionIpfsId'],
-  returnValueTest: {
-    comparator: '=',
-    value: '<YOUR_LIT_ACTION_IPFS_CID>',
-  },
-}
-```
+Decryption with Lit can be performed either client-side by an authorized user or within a Lit Action.
+
+Using decryption within a Lit Action is useful for performing operations over sensitive data, where the data itself remains private within the confines of each Lit node's Trusted Execution Environment (TEE). You can learn more about Lit's architecture [here](../../resources/how-it-works#sealed-and-confidential-hardware.md).
+
+Lit Actions have two methods for decrypting data: `decryptToSingleNode` and `decryptAndCombine`. The former reduces the execution scope of the Lit Action to a single node and decrypts the data there. The latter collects each Lit node's decryption share, combines them, and then decrypts the data on a single node. The key difference between the two is that `decryptToSingleNode` only uses the decryption share of a single Lit node, while `decryptAndCombine` uses all of the decryption shares of all Lit nodes.
+
+When `decryptToSingleNode` is used, the execution scope being reduced to a single Lit node means that any behavior that requires multiple nodes (i.e. console logs, `signAndCombineEcdsa`) will encounter a timeout error.
```

Comment:
When you say console logs, do you mean `console.log`? I don't think there's an affect on these when running within a single node?

Reply:
We should add a snippet on how to end execution on all the Lit node that weren't selected for the decryption

That looks something like [this](https://github.com/LIT-Protocol/js-sdk/blob/110ebd426a7a7910eaf6012cdba137f9f535cd40/packages/wrapped-keys/src/lib/litActions/ethereum/src/signMessageWithEthereumEncryptedKey.js#L17-L36)